capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -14,6 +14,13 @@ class ClubPage extends Component {
         .then(response => response.json()).then(res => this.setState({club: res[0]}));
   }
 
+  handleClick = (e) => {
+    e.preventDefault();
```

Comment:
Not sure this is necessary here. `preventDefault` is usually needed in methods that handle form submissions, since the default submit behavior is to redirect the browser. I don't think there's any click-handling default behavior you need to suppress though.

Reply:
I became so used to adding it to all my handlers, I didn't realize it was unneeded here. Will remove.